ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -176,7 +176,7 @@
 )
 
 skipif_bm = pytest.mark.skipif(
-    config.ENV_DATA["platform"].lower() == "baremetal"
+    (config.ENV_DATA["platform"].lower() == "baremetal")
```

Comment:
why was this changed added?

Reply:
Vasu, I added code and removed it, so the braces stayed. Let me remove the braces.